ext/js/display/search-display-controller.js

```diff
@@ -268,10 +268,9 @@ export class SearchDisplayController {
     }
 
     /** */
-    _onCopy() {
+    async _onCopy() {
```

Comment:
Why did you change this to async? I would've thought a function signature change would require changing the call sites of `_onCopy` as well

Reply:
_onCopy runs based off of an event listener. Event listener calls are essentially voids.